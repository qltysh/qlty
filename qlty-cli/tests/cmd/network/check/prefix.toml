bin.name = "qlty"
args = ["check", "--all", "--no-cache"]
status.code = 1
