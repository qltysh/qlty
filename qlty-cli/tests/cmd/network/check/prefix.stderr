✖ 2 issues
✖ 2 unformatted files
